Workade/Views&ViewModels/NearbyPlace/GalleryView.swift

```diff
@@ -8,8 +8,11 @@
 import UIKit
 
 class GalleryView: UIView {
-    
-    let bottomSafeArea = UIApplication.shared.windows.first?.safeAreaInsets.bottom ?? 44
+    private var bottomSafeArea: CGFloat {
+        guard let scene = UIApplication.shared.connectedScenes.first as? UIWindowScene else { return CGFloat(44) }
+        guard let window = scene.windows.first else { return CGFloat(44) }
+        return window.safeAreaInsets.top
+    }
```

Comment:
[p2] project 전체에 이와 같은 코드가 5군데 있었습니다. extension을 사용해서, 
```suggestion
    extension CGFloat {
    static var safeAreaTop: CGFloat {
        guard let scene = UIApplication.shared.connectedScenes.first as? UIWindowScene else { return CGFloat(44) }
        guard let window = scene.windows.first else { return CGFloat(44) }
        return window.safeAreaInsets.top
    }
}
```
가져다 쓸때는 .safeAreaTop을 사용하는 방식은 어떠신가요??

Reply:
[p2] 추가) guard문별로 return 값이 같으니 guard구문도 한 줄로 줄일 수 있을 듯 합니다.
```suggestion
    extension CGFloat {
    static var safeAreaTop: CGFloat {
        guard let scene = UIApplication.shared.connectedScenes.first as? UIWindowScene,
              let window = scene.windows.first else { return CGFloat(44) }
        return window.safeAreaInsets.top
    }
}
```

Reply:
해당 부분 별도의 브랜치를 따서 Extension 을 이용하여 구현하고 적용해 놓은 상태인데, 다음 스크럼때 어떻게 할지 정해지면 바로 적용할수 있을것 같습니다 :)